### DISCUSSION TITLE
How to add an item in a blog post's sidebar?

### DISCUSSION THREAD

**Discussion by CLOVIS-AI** [squidfunk/mkdocs-material#7819] [Q&A] [answered]
I have recently added support for RSS in my blog. I'd to like to emphasize this to readers, by adding a link to the RSS feed in the post's sidebar, alongside its existing metadata.

My understanding is that I should do this by overriding the [`blog-post.html` template](https://github.com/squidfunk/mkdocs-material/blob/51e4d4b212fc85a02fa8be49384b3f76a0db7237/src/templates/blog-post.html) to be able to add new information. However, to do this, I would need the sidebar to be in its own `block`, no? And it doesn't appear to be.

So, how can I add a new item in a post's sidebar?

**Comment by kamilkrzyskow** [comment, marked as answer]
Hello @CLOVIS-AI,
you're correct, there is no block for overrides defined in that template, the theme limits the blocks only to `base.html`.
Basically you have to copy the whole content of the template and modify the HTML to your liking, as it overrides the `container` block.
Name it `blog-post.html` so that your overridden variant will overwrite the file from the theme, during build.
